### ISSUE TITLE
[UI] Themes file input styling issue

### ISSUE THREAD

**Issue by laurielim** [mautic/mautic#14385]
### Mautic Series

6.0.x series

### Mautic installed version

6.0.0

### Way of installing

Other (please specify if relevant in the bug report)

### PHP version

8.3.14

### What browsers are you seeing the problem on?

Chrome

### What happened?

Was checking lists when I saw that the themes list action buttons were not showing properly, see image:
![image](https://github.com/user-attachments/assets/9a7dba83-c74e-49d7-b468-dd4b75d10da4)

### How can we reproduce this issue?

- Open 6.x on Gitpod 
- Go to the Themes page (/s/themes)
- See that the file input is hidden behind the install button

### Code of Conduct

- [X] I confirm that I have read and agree to follow this project's Code of Conduct

<br /><hr>
Care about this issue? Want to get it resolved sooner? If you are a <a href='https://www.mautic.org/become-a-member-of-mautic'>member of Mautic</a>, you can add some funds to the <a href='https://opencollective.com/mautic/projects/bounties'>Bounties Project</a> so that the person who completes this task can claim those funds once it is merged by a member of the core team! Read the docs <a href='https://contribute.mautic.org/product-team/mautic-bounty-programme'>here.</a>

**Comment by matbcvo** [comment]
Duplicate of #14172.
This has been fixed in the 5.2 branch and will be merged into the 6.x branch.
